react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -40,12 +39,14 @@ const Map: React.FC<MapProps> = ({
     );
 
     React.useEffect(() => {
+        console.log("use effect create store")
         store.current = createStore(deckglSpec, setSpecPatch);
     }, [setSpecPatch]);
 
     const [specObj, setSpecObj] = React.useState(null);
 
     React.useEffect(() => {
+        console.log("Use effect sepec obj update");
```

Comment:
This hook is not getting called when I do selection in the tool bar to hide a layer.